src/Mod/PartDesign/App/FeatureSketchBased.cpp

```diff
@@ -695,6 +699,57 @@ void ProfileBased::getUpToFaceFromLinkSub(TopoShape& upToFace, const App::Proper
     }
 }
 
+int ProfileBased::getUpToShapeFromLinkSubList(TopoShape& upToShape, const App::PropertyLinkSubList& refShape)
+{
+    int ret = 0;
+    bool hasShape = false;
+
+    auto subSets = refShape.getSubListValues();
+
+    std::vector<TopoShape> shapeList;
+
+    for (auto &subSet : subSets){
+        auto ref = subSet.first;
+        if (ref->isDerivedFrom<App::Plane>()) {
+            shapeList.push_back(makeShapeFromPlane(ref));
+            ret ++;
+        } else {
+            if (!ref->isDerivedFrom<Part::Feature>())
+                throw Base::TypeError("SketchBased: Must be face of a feature");
+
+
+            auto subStrings = subSet.second;
+            if (subStrings.empty() || subStrings[0].empty()) {
+                TopoShape baseShape = Part::Feature::getTopoShape(ref, nullptr, false);
+                shapeList.push_back(baseShape);
+                ret ++;
+                hasShape = true;
+            }
+            else {
+                for (auto &subString : subStrings){
+                    auto face = Part::Feature::getTopoShape(ref, subString.c_str(), false);
+                    if (face.isNull())
+                        throw Base::ValueError("SketchBased: Failed to extract face");
+                    shapeList.push_back(face);
+                    ret ++;
+                }
+            }
+        }
+    }
+    if (ret == 0){
+        throw Base::ValueError("SketchBased: No face selected");
+    }
+    if (ret == 1){
+        upToShape = shapeList[0];
+    }
+    else {
+        // create a unique shape with all selected ones
+        upToShape.makeElementCompound(shapeList);

```

Comment:
This compound is next used in makeElementPrismUntil.

Something is calling [BRepSweep_NumLinearRegularSweep](https://dev.opencascade.org/doc/refman/html/class_b_rep_sweep___num_linear_regular_sweep.html) which fails at
[BRepSweep_NumLinearRegularSweep.cxx#L235-L240](https://github.com/Open-Cascade-SAS/OCCT/blob/b78ccf1f9b69249d5f5f9c4c1e137f0f16ea49a7/src/BRepSweep/BRepSweep_NumLinearRegularSweep.cxx#L235-L240)
```cpp
      case TopAbs_SOLID :
        throw Standard_NoSuchObject("Solids are not Processed");
        break;
      case TopAbs_COMPSOLID :
        throw Standard_NoSuchObject("Solids are not Processed");
        break;
```
I'm stuck because I don't know how to create a breakpoint in an external library (I'm using gdb in QtCreator) in order to get the process state.

Reply:
FYI I had help in the qt-creator IRC channel to debug FC with a breakpoint on OCC.